opcua/server/history.py

```diff
@@ -38,7 +38,7 @@ def read_node_history(self, node_id, start, end, nb_values):
         """
         raise NotImplementedError
 
-    def new_historized_event(self, event, period):
+    def new_historized_event(self, source_id, etype, period):
```

Comment:
I propose 
`def new_historized_event(self, source_id, list_of_properties, period):`
But I have not thought much about it...